code/style.css

```diff
@@ -1,4 +1,170 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* General styling */
+body {
+  background: black;
+  color: whitesmoke;
+  font-family: 'Courier New', Courier, monospace;
+  
 }
+
+
+/* Header */
+header img {
+  height: 50px;
+  width: 50px;
+}
+header {
+  background:black;
+  padding: 5px;
+  border: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+.hamburger {
+  display: flex;
+  height: 25px;
+  width: 25px;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+.hamburger div {
+  height: 4px;
+  width: 25px;
+  background: whitesmoke;
+
+}
+
+nav {
+   display: none; 
+}
+
+nav a {
+  text-decoration: none;
+  color: whitesmoke;
+  margin-left: 10px;
+
+}
+
+
+
+
+/* Big news */
+
+.big-news-container {
+  padding: 100px 20px;
+  display: flex;
+  flex-direction: column;
+
+}
+
+.big-news-container img {
+  width: 100%;
+}
+
+
+/* Card grid  */
+.card-container {
+  display: flex;
+  justify-content: center;
+
+  }
+
+   .card-container img {
+  width: 90%; 
+  height: 90%;
+  }
+
+  .cards {
+    max-width: 100%;
+    margin: 0px;
+    display: grid;
+    grid-row-gap: 1rem;
+    
+  }
+  .card {
+    display: flex;
+    align-items: column;
+    flex-wrap: wrap;
+    background-color: whitesmoke;
+    color: gray;
+    padding: 2rem;
+    margin: 2rem;
+    margin-right: 1rem;
+    height: 15rem;
+    width: 15rem;
```

Comment:
If you take a look at how your page looks when it has just reached the desktop media query, you can see that the fixed width of the cards is creating a horizontal scroll. To avoid that, you can
- remove the `card-container` with flexbox and just let the grid do its thing
- remove the fixed width

This will force all the cards to only take up the space of one column. See this codepen example: https://codepen.io/hippiekick/pen/ExKwvOr?editors=1100